html/elements/a.json

```diff
@@ -354,7 +354,7 @@
               "safari_ios": "mirror",
               "samsunginternet_android": "mirror",
               "webview_android": {
-                "version_added": false
+                "version_added": "88"
               }
```

Comment:
see the workflow run https://github.com/mdn/browser-compat-data/actions/runs/9064486022/job/24902841850?pr=23076

![image](https://github.com/mdn/browser-compat-data/assets/95597335/9212516e-718d-47d0-bbfd-e4f9fb042898)

Reply:
@skyclouds2001 Fixed. :)

Reply:
Or not... sec. 🫣